codalab/lib/bundle_cli.py

```diff
@@ -1730,6 +1732,29 @@ def wrap(string):
                 self.print_target_info(client, (bundle_uuid, item['name']), decorate=True)
 
     @Commands.command(
+        'mount',
+        help=[
+            'Mount the contents of a bundle at a read-only mountpoint.',
+        ],
+        arguments=(
+            Commands.Argument('target_spec', help=TARGET_SPEC_FORMAT, completer=TargetsCompleter),
+            Commands.Argument('--mountpoint', help='Empty directory path to set up as the mountpoint for FUSE.'),
```

Comment:
I am hesitant to create the mountpoint for the user, because the mountpoint itself will need to have escalated permissions as a FUSE call is a system call. It will be risky to try to hide these intricacies from the user even if we are fine with adding the necessary bloat to the code to add a bunch of checks